lib/mustache/generator.rb

```diff
@@ -152,6 +152,37 @@ def on_partial(name, offset, indentation)
       ev("ctx.partial(#{name.to_sym.inspect}, #{indentation.inspect})")
     end
 
+    # Fired when the compiler finds a inheritance. We want to return code
+    # which stores the block variables in this template and  
+    # calls the parent at runtime.
+    def on_parent(name, offset, content, raw, delims)
+      result = ""  
+      parent_name = name[2].first  
+      content.each { |x| 
+        case 
+	  when x != :multi && x[0] === :mustache && x[1] === :blockvar
```

Comment:
`when` should be at the same level of indentation as `case`.
If there's only one condition, why did you opt for the `case` instead of an `if` condition? Just curious.